src/lib.rs

```diff
@@ -229,6 +234,22 @@ impl GlutinWindow {
         }
     }
 
+    fn poll_events(&mut self) {
```

Comment:
This function doesn't seem to be working as intended; my application loops forever here.

Reply:
This is fixed now.